frontend/console/src/features/timeline/timeline.utils.ts

```diff
@@ -0,0 +1,23 @@
+import type { Event } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+
+const eventBackgroundColorMap: Record<string, string> = {
+  log: 'bg-gray-500',
+  call: 'bg-indigo-500',
```

Comment:
What do you think about making this one different from the deployment event colors? Possibly even just a different shade of indigo?

Reply:
Honestly, I'm not sure :) I like the idea of each thing having its own color, but it also starts to get very colorful really quick! I'll give it a shot to see what we think. As I start to think about all the other things like `pubsub`, `db`, etc. we might end up with like "theme" colors so they are somehow connected to each other.

Reply:
I'll go with this light green for now and we can adjust if we wanna.

![Screenshot 2024-09-24 at 11 54 32 AM](https://github.com/user-attachments/assets/ef819ad6-1f5c-434a-aa85-128e88083af9)